tests/Dfc.CourseDirectory.Testing/TestData/TestData.CreateLearningAimRef.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Threading.Tasks;
+using Dfc.CourseDirectory.Core.DataStore.Sql.Queries;
+
+namespace Dfc.CourseDirectory.Testing
+{
+    public partial class TestData
+    {
+        public Task<string> CreateLearningAimRef() => WithSqlQueryDispatcher(async dispatcher =>
+        {
+            var learnAimRef = new Random().Next(100000, 109999).ToString("D8");
+
+            await dispatcher.ExecuteQuery(new UpsertLarsLearningDeliveries()
+            {
+                Records = new[]
+                {
+                    new UpsertLarsLearningDeliveriesRecord()
+                    {
+                        LearnAimRef = learnAimRef,
+
+                        // Unused columns that have `not null` constraints :-/
```

Comment:
This junk is here temporarily until we sort our LARS import out to not blindly import the source CSVs